app/javascript/packages/form-steps/form-steps.tsx

```diff
@@ -171,6 +171,11 @@ interface FormStepsProps {
    */
   basePath?: string;
 
+  /**
+   * URL for reset password page in rails used for redirect
+   */
+  resetPasswordUrl?: string;
```

Comment:
This package is also used in document capture, where "reset password" doesn't apply. This package should be kept neutral to whichever implementation a form flow might be used for. I think we should create a new context.

Maybe "VerifyFlowContext" for common verify flow values (also applies below for `basePath`).